lib/getPostcssResult.js

```diff
@@ -19,7 +20,7 @@ module.exports = function (stylelint/*: stylelint$internalApi*/)/*: Promise<?Obj
     customSyntax?: string
   }*/ = arguments.length > 1 && arguments[1] !== undefined ? arguments[1] : {}
 
-  const cached = stylelint._postcssResultCache.get(options.filePath)
+  const cached/*: ?Object*/ = stylelint._postcssResultCache.get(options.filePath)
```

Comment:
Oh, whoops ... this is the one you emailed about! You proposed a type like this, right?

```
export type postcss$result = {
  propcessor: any,
  messages: <Array>,
  root: Object,
  opts: Object,
  css: string,
  map: any,
}
```

Let's start with just this: 

```
export type postcss$result = {
  root: Object
  css: string
}
```

And see if that works. We can add more of the Result's properties as needed.